profile/CSIP.xml

```diff
@@ -1203,8 +1202,8 @@
             </requirement>
             <requirement ID="CSIP103" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel" EXAMPLES="structMapExample1">
                 <description>
-                    <head>File division label</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The file division (div) element in the package uses the value "Representations" as the value for the attribute LABEL.</p>
+                    <head>Content division label</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The content division (div) element in the package uses the value "Representations" from the vocabulary as the value for the attribute LABEL.</p>
```

Comment:
The package's content division (div) element must have the `@LABEL` attribute value "Representations", taken from the vocabulary.